circle.yml

```diff
@@ -27,7 +27,6 @@ jobs:
       - run: 
           name: build with open-wc
           command: npm run build:owc;
-      - run: npm run test;
```

Comment:
moved the pipeline work into [WEB-267](https://bulibrary.atlassian.net/browse/WEB-267)